internal/core/data/init.go

```diff
@@ -116,3 +129,23 @@ func (b *Bootstrap) BootstrapHandler(_ context.Context, _ *sync.WaitGroup, _ sta
 
 	return true
 }
+
+// loadV2Routes creates a new command-query router and handles the related mux.Router initialization for API V2 routes.
+func (b *Bootstrap) loadV2Routes(dic *di.Container, lc logger.LoggingClient) {
```

Comment:
Other than the receiver, this method is identical across all the services in this PR. Perhaps this could be collapsed into a common function?

Reply:
It's identical currently because there are no service-specific controllers being added.  

Longer-term vision is here -- https://github.com/michaelestrin/edgex-go/blob/issue-2277/internal/pkg/v2/README.md

> Single Configuration-Driven Executable
The design supports a single executable capable of supporting endpoints from one or more of today's existing services. This could be hardcoded -- as it is in the current Fuji release. It could also easily be configuration-driven -- a single executable could provide any or all existing functionality for all core, support, and system management services based on a configuration file. This would simplify artifact creation and provide a more flexible deployment story.

This would reduce code duplication across services significantly.

Regardless, there is still duplication -- like this -- in different places in the PR.  I'm evolving the design, identifying the duplication, and extracting (as I did within the last day or two with acceptance tests and creating test.Case) as it makes sense.  It doesn't make sense yet for me to extract this common code into one place.  It probably will after a few services have their controllers implemented and the full pattern becomes apparent.